ui/apps/platform/src/Containers/Vulnerabilities/components/AdvancedFiltersToolbar.tsx

```diff
@@ -61,6 +61,7 @@ type AdvancedFiltersToolbarProps = {
     searchFilterConfig: CompoundSearchFilterProps['config'];
     searchFilter: SearchFilter;
     onFilterChange: (searchFilter: SearchFilter, payload: OnSearchPayload) => void;
+    cveStatusFilterField?: 'FIXABLE' | 'CLUSTER CVE FIXABLE';
```

Comment:
Leaving this optional, with a default of "FIXABLE". The primary use case is for this field, with "Cluster CVE Fixable" being the exception, so I didn't feel the need to update every usage of this component throughout VM.